.github/ISSUE_TEMPLATE/security.yml

```
@@ -0,0 +1,95 @@
+name: Security Report
+description: Report a security vulnerability (Please read SECURITY.md first)
+title: "[SECURITY] "
+labels: ["security", "critical"]
+assignees: ["irfandi"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ **IMPORTANT**: For security vulnerabilities, please **DO NOT** create public issues.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)